### ISSUE TITLE
Disc Priest: atonement healing source crash

### ISSUE THREAD

**Issue by MartijnHols** [WoWAnalyzer/WoWAnalyzer#788]
https://sentry.io/share/issue/33ce182b00b64e65843b0c403b2b69c5/
https://sentry.io/share/issue/c19dd05d2a494afda3c444cf201142a8/

```
TypeError: Cannot read property 'ability' of null
  at _addHealing (Parser/Priest/Discipline/Modules/Features/AtonementHealingDone.js:31:28)
  at apply (Parser/Priest/Discipline/Modules/Features/AtonementHealingDone.js:27:10)
  at apply (Parser/Core/Analyzer.js:29:7)
  at apply (Parser/Core/Analyzer.js:11:7)
  at None (Parser/Core/CombatLogParser.js:368:11)
...
(11 additional frame(s) were not displayed)
```

https://wowanalyzer.com/report/MqtdTJH6mWbAyYfg/2-Normal+Goroth+-+Kill+(3:42)/Laiza

**Comment by karlpralow** [comment]
This issue comes from the log having 2 discpline priests. I'll fix this when I get home

**Comment by karlpralow** [comment]
Fixed in PR #804
